os/compression/Makefile

```diff
@@ -34,6 +37,10 @@ ifeq ($(CONFIG_BUILD_PROTECTED),y)
 ifeq ($(CONFIG_COMPRESSION_TYPE),1)
 CFLAGS += -D_7ZIP_ST
 COMPRESSION_CSRCS += $(wildcard ./$(LZMA_SRCDIR)/*.c)
+else
+ifeq ($(CONFIG_COMPRESSION_TYPE),2)
```

Comment:
How about using definitions instead of 1 and 2?
```
in Makefile
LZMA=1
MINIZ=2

CFLAGS=+DLZMA +DMINIZ

in c and h files

#if CONFIG_COMPRESSION_TYPE == LZMA
#elif CONFIG... == MINIZ
```

Reply:
Ok. Will make these changes everywhere.

Reply:
Ok. Will make these changes everywhere.

Reply:
Done. Made changes everywhere in code.
Made a separate commit for it.